code/modules/hydroponics/plant_genes.dm

```diff
@@ -99,6 +99,7 @@
 	name = "Nutriment"
 	var/reagent_id = "nutriment"
 	var/rate = 0.04
+	var/list/blacklisted_reagents = list("potassium", "cornoil", "diethylamine", "stable_plasma", "welding_fuel", "glycerol")
```

Comment:
@XDTM this prevents insertion, bananas still keep their potassium, but it can't be inserted into another plant. the alternative is hoylwater/water blacklisting because that's an explosive recipe.